frontend/public/components/utils/dropdown.jsx

```diff
@@ -598,6 +532,43 @@ export class Dropdown extends DropdownMixin {
   }
 }
 
+export const Dropdown = (props) => {
+  const { storageKey } = props;
+
+  // Should be undefined so that we don't save undefined-bookmarks.
+  const bookmarkStorageKey = storageKey ? `${storageKey}-bookmarks` : undefined;
+
+  const [favoriteKey, setFavoriteKey] = useUserSettingsCompatibility(
+    storageKey,
+    storageKey,
+    null,
+    // TODO: enable sync when sync PR is merged
+  );
+  const [bookmarks, setBookmarks] = useUserSettingsCompatibility(
+    bookmarkStorageKey,
+    bookmarkStorageKey,
+    {},
```

Comment:
Can this be `undefined` to start?

Reply:
Yeah works fine. Removed the default so its now undefined.